src/gen/parser.ts

```diff
@@ -66,7 +66,10 @@ function parseName(
   const maybeOptional = /\[(.+)\]/.exec(input);
   const optional = maybeOptional && maybeOptional?.length > 1 ? true : false;
   const rawName = optional && maybeOptional ? maybeOptional[1] : input;
-  const name = rawName.replace(/^[^a-zA-Z_$]|[^0-9a-zA-Z_$]/g, '_');
+  const name = rawName
+    .replace(/^[^a-zA-Z_$]|[^0-9a-zA-Z_$]/g, '_')
+    // Replace reserved keyword `var`
+    .replace('var', 'v');
```

Comment:
Seems like a reasonable solution. The variable name doesn't appear to be important to the context of its function in the calls that use var as a parameter name.